doc/developer.texinfo

```diff
@@ -2446,27 +2436,150 @@ GNU Texinfo Manual (@uref{http://www.gnu.org/software/texinfo/manual/texinfo/tex
+@end example
+
+UNDONE: need to check doc/CHANGES time stamp.
+
+[Time: moments]
+
 
 @need 1000
+@item
+Set the version number in file @file{$DAIKONDIR/doc/VERSION}
+to an even number.
+You may do this manually, or if the value just needs to be incremented
+by 1 (to go from odd to even) you may use:
+
+@c "cd $DAIKONDIR" because the previous commands changed the working directory
+@example
+make update-dist-version-file
+@end example
+
+Optionally, override the distribution date (default: today) by
+redefining the environment variable @command{TODAY}:
+
+@example
+export TODAY='November 1, 2013'
+@end example
+
+Now, update the appropriate files with the date and version number
+of the release.
+
+@example
+make update-doc-dist-date-and-version
+@end example
+
+[Time: moments]
+
+UNDONE: we should combine and automate the step above and the
+step below.
+
+@need 1000
+@item
+Commit the version number changes back to the repository:
+
 @example
 cd $DAIKONDIR
-\rm -f `find . -type f -name '*~'`
+DAIKONVERSION=`cat $DAIKONDIR/doc/VERSION | xargs echo -n`
+git commit -a -m "Change version to $DAIKONVERSION"
+git push
+cd fjalar
+git commit -a -m "Change version to $DAIKONVERSION"
+git push
+@end example
+
+[Time: moments]
+
+
+@need 1000
+@item
+It is important to build the release candidate from a clean enlistment.
+This is to ensure we are testing only the files intended for release and
+that those files are the same as a user would receive with a clone of
+the repository. It is also recommend doing this step on your system's
+@file{scratch} or @file{tmp} directory. (@file{tmp} is used in the
+example commands below.)
+
+@example
+mkdir -p /tmp/$USER
+rm -rf /tmp/$USER/daikon
+git clone https://github.com/codespecs/daikon /tmp/$USER/daikon
+@end example
+
+[Time: 1 minute]
+
+
+@need 1000
+@item
+You now need to create a separate, new Bash command shell and set the 
+appropriate environment variables.
+The following commands may need to be modified
+for your particular system.
+
+@example
+cd /tmp/$USER/daikon
+export DAIKONDIR=`pwd`
+export JAVA_HOME=/homes/gws/markro/jdk7
+source $DAIKONDIR/scripts/daikon.bashrc
+@end example
+
+[Time: moments]
+
+@noindent
+Each of the steps below assumes that the current working directory is
+@file{$DAIKONDIR}.
+
+
+@need 1000
+@item
+Build everything to be distributed and copy it to the temporary staging area
+@c not @uref{...} because we don't want robots to notice and crawl the link
+(http://plse.cs.washington.edu/staging-daikon/).
+
+@example
 make staging
 @end example
 
+@noindent
+If the command immediately stops with the error message:
+@example
+** doc/CHANGES file is not up-to-date with respect to doc files.
+@end example
+@noindent
+You may simply:
+@example
+touch doc/CHANGES
+@end example
+
+The final output of this step
+will be a list of files that were added/removed since the
+last release.
+If any of these differences is unexpected, then investigate and correct the
```

Comment:
I suggest that it be one step, starting with creating a new shell and doing all the work in the new shell.  The first step of that would be creating the temporary directory, deleting the daikon subdirectory, and cloning.